mappings/net/minecraft/structure/Structure.mapping

```diff
@@ -6,22 +6,115 @@ CLASS net/minecraft/class_3499 net/minecraft/structure/Structure
 	METHOD method_15160 getSize ()Lnet/minecraft/class_2338;
 	METHOD method_15161 setAuthor (Ljava/lang/String;)V
 		ARG 1 name
+	METHOD method_15162 mirrorPos (Lnet/minecraft/class_2338;Lnet/minecraft/class_2415;Lnet/minecraft/class_2470;II)Lnet/minecraft/class_2338;
```

Comment:
What does this do exactly? Does it just mirror the structure? If yes, it should just be `mirror`. Otherwise, what do you mean by "mirroring the position"?